Makefile

```diff
@@ -246,6 +246,17 @@ test-postgres: tester
 tests: test #alias for test
 .PHONY: tests
 
+test-cephs3: tester
+	@echo "\033[1;34mRunning tests with Postgres.\033[0m"
+	@$(call create_docker_network)
+	@$(call run_postgres)
+	@echo "\033[1;34mRunning tests\033[0m"
+	$(CONTAINER_ENGINE) run $(CPUSET) --network noobaa-net --name noobaa_$(GIT_COMMIT)_$(NAME_POSTFIX) --env "SUPPRESS_LOGS=$(SUPPRESS_LOGS)" --env "POSTGRES_HOST=coretest-postgres-$(GIT_COMMIT)-$(NAME_POSTFIX)" --env "POSTGRES_USER=noobaa" --env "DB_TYPE=postgres" --env "POSTGRES_DBNAME=coretest" -v $(PWD)/logs:/logs $(TESTER_TAG) "./src/test/system_tests/ceph_s3_tests/run_ceph_test_on_test_container.sh" || true
```

Comment:
why is it `|| true` ? if that fails it will mark the passing tests as green.

Reply:
You are right. I forgot to remove this, I was running it locally and wanted to remove the containers after the test finishes regardless of the results. 

Removed ✅